app/forms/candidate_interface/degree_completion_status_form.rb

```diff
@@ -0,0 +1,26 @@
+module CandidateInterface
+  class DegreeCompletionStatusForm
+    include ActiveModel::Model
+
+    attr_accessor :degree_completed
+
+    validates :degree_completed, presence: true
+
+    def save(degree)
+      return false unless valid?
+
+      degree.update!(predicted_grade: grade_is_predicted?)
+    end
+
+    def fill(degree)
+      self.degree_completed = degree.completed? ? 'yes' : 'no'
```

Comment:
Went with this because I thought it read clearer on that line, but I think you're right - symmetry between `fill` and `save` would be less confusing. Will update!